www/source/docs/run-packages-overview.html.md.erb

```diff
@@ -17,6 +17,8 @@ When entering an interactive studio, a supervisor is started for you in the back
 
 Because the supervisor is running in the background, you will not see the supervisor output as you start your service. However you can use the `slog` (supervisor log) command that will stream the tail of the supervisor output.
 
+You can use the `HAB_DOCKER_OPTS` ENV variable to pass flags to the docker run that starts the studio. e.g. `HAB_DOCKER_OPTS="-p 8080:8080"` to expose port 8080 for the service you are running inside the studio out to your controlling host.
```

Comment:
Might want to note that this only works under Mac dev environment and this is only for hab studio work.  I'm not sure about Windows dev environment, but I'm guessing that studio for Windows is also docker so which means this may apply to them too.

But to reiterate this is ONLY needed for hab studio work.  The container that is exported out of `hab pkg export docker <pkg_ident>` is just a normal docker container and you would expose the ports normally just not with HAB_DOCKER_OPTS.